cmd/buildah/rename.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"github.com/pkg/errors"
+	"github.com/projectatomic/buildah"
+	"github.com/urfave/cli"
+)
+
+var (
+	renameDescription = "Rename a local container"
+	renameCommand     = cli.Command{
+		Name:           "rename",
+		Usage:          "Rename a container",
+		Description:    renameDescription,
+		Action:         renameCmd,
+		ArgsUsage:      "CONTAINER-NAME-OR-ID CONTAINER-NAME",
+		SkipArgReorder: true,
+	}
+)
+
+func renameCmd(c *cli.Context) error {
+	var builder *buildah.Builder
+
+	args := c.Args()
+	if len(args) < 2 {
```

Comment:
Nit: Could simplify this code to say
if len(args) == 2 {
          return errors.Errorf("container and it's new name must be specified'")
}

Reply:
s/b '!=2` no?  Or are the cold meds kicking in for me?